src/plugins/markdown-with-image/markdown-with-image.yml

```diff
@@ -13,6 +13,9 @@ imageUrl:
 imageWidth:
   en: Image width
   de: Bildbreite
+imageWidthInfo:
+  en: Represents the width taken by the image, from the total allocated space
+  de: Represents the width taken by the image, from the total allocated space [but in German]
```

Comment:
Repräsentiert die Breite, die das Bild einnimmt, gemessen an der Gesamtbreite des Abschnitts

Reply:
Also in German, but better! Thanks 🙏 😂